app/presenters/development_presenter.rb

```diff
@@ -105,4 +111,18 @@ def short_address
     "#{item.address}, #{item.city}"
   end
 
+  def category_attributes(category)
+    item.attributes.select { |k, v|
+      categorized_attributes.fetch(category, {}).include?(k.to_sym) && !v.nil?
+    }
+  end
+
+  def categorized_attributes
+    { physical:   [:tothu, :commsf, :prjarea, :stories, :height],
+      housing:    [:singfamhu, :twnhsmmult, :lgmultifam, :tothu],
+      commercial: [:fa_ret, :fa_ofcmd, :fa_indmf, :fa_whs, :fa_rnd,
+        :fa_edinst, :fa_other, :fa_hotel] }
```

Comment:
Hard-coded groupings can be dangerous -- might consider refactoring later when there's time.